CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On Android, calling `WindowEvent::Focused` now works properly instead of always returning false. 
+- On Windows, fix borderless fullscreen alt-tab behaviour to no longer be always on top.
```

Comment:
I feel like it should be 'change', and not 'fix'?

```suggestion
- On Windows, fix borderless fullscreen alt-tab behaviour to no longer be always on top.
```

Reply:
It's the _alt-tab behaviour_ that is being fixed. I've clarified it.

Reply:
With a new wording it makes a bit more sense as 'fixed', thx.